DifferenceList/monad

```diff
@@ -0,0 +1,26 @@
+    let fromPureBind = ./../Monad/fromPureBind
+
+in  let DifferenceList = ./Type
+
+in  let append = ./append
+
+in  let empty = ./empty
+
+in  let foldable = ./foldable
+
+in  let singleton = ./singleton
+
+in  fromPureBind
+    DifferenceList
+    (λ(a : Type) → singleton a)
```

Comment:
If you _don’t_ inline `singleton` here, this should at least just be `singleton` instead of applying the first argument.